buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -57,13 +57,13 @@ object Versions {
         const val runPaper = "2.2.0"
         const val paperWeight = "1.5.6"
     }
-    
-    object Sponge {
-        const val sponge = "9.0.0-SNAPSHOT"
-        const val mixin = "0.8.2"
-        const val minecraft = "1.17.1"
-    }
-    
+//
+//    object Sponge {
```

Comment:
Why is sponge commented out in a PR relating to quilt?

Reply:
just minor clean up